### PR TITLE
adding video output for block content

### PR DESCRIPTION
**Pull request by readeral**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -110,6 +113,7 @@
     "react-feather": "^2.0.3",
     "react-select": "^3.0.8",
     "react-stripe-elements": "^6.0.1",
+    "react-youtube": "^7.9.0",
```
Comment (BarryThePenguin) [inline package.json]: Why not https://github.com/u-wave/react-youtube ?

Comment (BarryThePenguin) [reply package.json]: Looking at the u-wave players, we can just pass in video urls.
That way we don't need to add `get-video-id` too 🎉

Comment (readeral) [reply package.json]: wicked

Comment (readeral) [reply package.json]: Mmm passing urls seems the case for vimeo, but not for youtube, see here (youtube) https://github.com/u-wave/react-youtube/blob/master/src/index.js#L246 vs here (vimeo) https://github.com/u-wave/react-vimeo/blob/master/src/index.js#L166

Comment (BarryThePenguin) [reply package.json]: Ahk.. the youtube one says

> An 11-character string representing a YouTube video ID..

Comment (readeral) [reply package.json]: Which is a bummer, and might be worth making a pull request on the library to offer the option of the url, but in the meantime, we'll go with get-video-id

**Pull request merged**